website/docs/r/nat_gateway.html.markdown

```diff
@@ -52,6 +52,8 @@ The following arguments are supported:
 * `name` - (Optional) Name of the nat gateway. The value can have a string of 2 to 128 characters, must contain only alphanumeric characters or hyphens, such as "-",".","_", and must not begin or end with a hyphen, and must not begin with http:// or https://. Defaults to null.
 * `description` - (Optional) Description of the nat gateway, This description can have a string of 2 to 256 characters, It cannot begin with http:// or https://. Defaults to null.
 * `bandwidth_packages` - (Optional) A list of bandwidth packages for the nat gatway. Only support nat gateway created before 00:00 on November 4, 2017. Available in v1.13.0+ and v1.7.1-.
+* `instance_charge_type` - (Optional, ForceNew) The billing method of the nat gateway. Valid values are "PrePaid" and "PostPaid". Default to "PostPaid".
```

Comment:
Please modify `Optional, ForceNew` to `Optional, ForceNew, Available in 1.39.0+`